src/Console/Commands/DefaultCommand.php

```diff
@@ -31,7 +32,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $kernel = Kernel::default();
 
         $issues = $kernel->handle([
-            'directory' => $directory = $this->inferProjectPath(),
+            'directory' => $directory = $this->findPathToScan($input),
```

Comment:
This change looks good, but it means that the pull request adds two features (which isn't bad, it just means @nunomaduro can either decide to take both or none).

I would either rename the PR to show you do both or create a new clean branch and re-commit the changes on their own. That way it can be decided for each feature, what gets merged.